src/client/components/ProfileFormComponent/ProfileFormComponent.stories.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import Avatar from './ProfileFormComponent';
+import avatarImage from '../../assets/images/avatar.png';
+import PropTypes from 'prop-types';
+
+export default {
+  title: 'Avatar',
+  component: Avatar,
+  argTypes: {
+    color: {
+      control: { type: 'select', options: ['#DB643D', '#535C75', '#000000'] },
+    },
+    width: {
+      control: { type: 'range', min: 400, max: 1200, step: 50 },
+    },
+    fontFamily: {
+      control: {
+        type: 'radio',
+        options: ['Roboto', 'Roboto Mono'],
+      },
+    },
+  },
+};
+
+const Template = ({
+  color,
+  width,
+  fontFamily,
+  userName,
+  userEmail,
+  userPassword,
+  userImage,
+}) => (
+  <Avatar
```

Comment:
here i can see this :   Property 'onChange' is missing in type '{ color: any; width: any; fontFamily: any; userName: any; userEmail: any; userPassword: any; userImage: any; }' but required in type 'Pick<Pick<{ color: any; width: any; fontFamily: any; userName: any; userEmail: any; userPassword: any; onChange: any; userImage: any; }, never> & Pick<InferProps<{ color: Requireable<string>; width: Validator<...>; fontFamily: Requireable<...>; }>, "color" | ... 1 more ... | "fontFamily"> & Pick<...>, "width" | ... ...'.
(If it is warning you can ignore)

Reply:
because i was not added those variables in proptypes , i will do that thing, thank you.